### PR TITLE
Added a TimeSeries Plot Example

### PR DESCRIPTION
**Pull request by abotiamnot**
An example with a few important functions available in TimeSeries

### CODE REVIEW

**Comment by dstansby** [comment]
I think it should be more like this example, where the data is plotted: http://docs.heliopy.org/en/stable/auto_examples/plot_importing_data.html#sphx-glr-auto-examples-plot-importing-data-py

I'm not too keen on using `peek()`, since it's hard to then customise a plot when it has been created.

File: examples/plot_timeseries.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import matplotlib.pyplot as plt
+from datetime import datetime
+
+starttime = datetime(1993, 1, 1, 0, 0, 0)
+endtime = datetime(1993, 2, 1, 0, 0, 0)
+
+timeseries_data = ulysses.swics_abundances(starttime, endtime)
+
+# timeseries_data is a TimeSeries data type
+# Using the timeseries.data function, one can obtain the Pandas DataFrame of the data
```
Comment (abotiamnot) [inline examples/plot_timeseries.py]: I think this is the most important point of using TimeSeries. Ease of Pandas DataFrame plus units!

File: examples/plot_timeseries.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+print(timeseries_data.data.keys())
+fig, axs = plt.subplots(2, 1, sharex=True)
+axs[0].plot(timeseries_data.data['VEL_ALPHA'])
+axs[1].plot(timeseries_data.data['RAT_C6_C5'])
+axs[1].plot(timeseries_data.data['RAT_O7_O6'])
+axs[1].plot(timeseries_data.data['RAT_FE_O'])
+
```
Comment (dstansby) [inline examples/plot_timeseries.py]: Can we get a `axs[1].set_yscale('log')` here? It's just a convention for the specific data that has been plotted in the second axes!

File: examples/plot_timeseries.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+timeseries_data = ulysses.swics_abundances(starttime, endtime)
+
+# timeseries_data is a TimeSeries data type
+# Using the timeseries.data function, one can obtain the Pandas DataFrame of the data
+
+print(timeseries_data.data.keys())
+fig, axs = plt.subplots(2, 1, sharex=True)
+axs[0].plot(timeseries_data.data['VEL_ALPHA'])
+axs[1].set_yscale('log')
```
Comment (abotiamnot) [inline examples/plot_timeseries.py]: Is this the correct method to do it?

**Pull request merged**